### PR TITLE
Image asset update

### PR DESCRIPTION
**Pull request by MonsieurMantis**
I updated the image assets and cat generation

### CODE REVIEW

File: Backend/cat.js [hunk 1]
```diff
@@ -1,20 +1,25 @@
 import words from './generic-food.js'
 
 // Ordered by rarity. Least rare first.
-const CAT_EYES = ["orange", "green", "blue", "gold"];
+const CAT_EYES_LEFT = ["orangeLeft", "greenLeft", "blueLeft", "goldLeft"];
+const CAT_EYES_RIGHT = ["orangeRight", "greenRight", "blueRight", "goldRight"];
 const CAT_COLORS = ["orange", "brown", "white", "tabby", "tshell", "calico", "black"];
-const CAT_HATS = ["partyBlue", "bowler", "cowboy", "partyPurple", "wizardBlue", "goggles", "cowboyPink", "wizardPurple", "partyEarth", "crown"];
+const CAT_HATS = ["partyBlue", "bowler", "brownHair", "cowboy", "partyPurple", "blondeHair", "wizardBlue", "goggles", "cowboyPink", "gingerHair", "wizardPurple", "partyEarth", "rainbow", "crown"];
```
Comment (kthchew) [inline Backend/cat.js:7]: Should these match the capitalization of the image file names?

File: Backend/cat.js [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
   // according to a random distribution depending on the rarity the box should have.
   constructor(rarityGenerator) {
-    const eyeIndex = rarityGenerator(CAT_EYES.length - 1);
+    let eyeIndex = rarityGenerator(CAT_EYES_LEFT.length - 1);
     const colorIndex = rarityGenerator(CAT_COLORS.length - 1);
     const hatIndex = rarityGenerator(CAT_HATS.length - 1);
 
-    this.eyes = CAT_EYES[eyeIndex];
+    this.leftEye = CAT_EYES_LEFT[eyeIndex];
+    if (Math.random()>0.995) {
```
Comment (kthchew) [inline Backend/cat.js:19]:
```suggestion
    if (Math.random()<0.005) {
```

for clarity

File: Backend/cat.js [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
+      eyeIndex = rarityGenerator(CAT_EYES_RIGHT.length - 1);
```
Comment (kthchew) [inline Backend/cat.js:20]: Maybe one way you could handle the rarity is to add both the left and right indices to the rarity?

**Review by MaximilianMeiler — APPROVED** [review]
LGTM

**Comment by MaximilianMeiler** [comment]
NOTE: this will break cats from ALL PREVIOUSLY USED accounts.
Developers: make sure you have new ones!

**Pull request merged**